spring-cloud-netflix-sidecar/src/test/java/org/springframework/cloud/netflix/sidecar/SidecarApplicationTests.java

```diff
@@ -27,22 +27,60 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertThat;
 
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = SidecarApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT, value = {
-		"spring.application.name=mytest", "spring.cloud.client.hostname=mhhost",
-		"spring.application.instance_id=1", "eureka.instance.hostname=mhhost",
-		"sidecar.port=7000" })
 public class SidecarApplicationTests {
 
-	@Autowired
-	EurekaInstanceConfigBean config;
+	@RunWith(SpringJUnit4ClassRunner.class)
+	@SpringBootTest(classes = SidecarApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT, value = {
+			"spring.application.name=mytest", "spring.cloud.client.hostname=mhhost",
```

Comment:
`spring.cloud.client.hostname` doesn't map to any config. My guess is these tests work find without it.